backend/src/main/java/com/huellapositiva/infrastructure/orm/entities/EmailRecoveryPassword.java

```diff
@@ -0,0 +1,39 @@
+package com.huellapositiva.infrastructure.orm.entities;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+@Entity
+@Table(name = "email_recovery_password")
+@Getter
+@Setter
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class EmailRecoveryPassword implements Serializable{
```

Comment:
Serializable {

I have seen it many times in this PR. Missing space before curly brace as per the naming convention.

Reply:
Fixed it.